src/v/cloud_storage/remote_probe.cc

```diff
@@ -129,6 +129,20 @@ remote_probe::remote_probe(
               "spillover_manifest_downloads",
               [this] { return get_spillover_manifest_downloads(); },
               sm::description("Number of spillover manifest downloads")),
+            sm::make_counter(
+              "controller_snapshot_successful_uploads",
+              [this] { return get_controller_snapshot_successful_uploads(); },
```

Comment:
question: would it make sense to graduate any of these metric to `public_metrics`? Can be done in a follow up.

Reply:
Hm, these feel a bit too internal/implementation-focused to be public. I'll leave them as is, but good point I'll think more about what metrics should be public in general for metadata uploads.